dev-docs/source/Standards/ReleaseNotesGuide.rst

```diff
@@ -0,0 +1,232 @@
+
+
+Adding release notes
+--------------------
+Release notes are added as new files in a pull request to avoid merge conflicts. Automated scripts then amalgamate these into the release notes available for users. If your work requires more than one note for the same subheading then it is fine to put them all in the same file. However,
+**new release notes should not be added to any existing files**.
+
+Amending release notes
+----------------------
+If you need to amend a release note from a previous Pull Request (PR) during sprint it is fine to do this as the automated script will update the release note. Provided developers use the naming conventions for files outlined below it should be easy to find the release note you wish to amend.
+If the release note you wish to amend is within a ``Used`` directory you will need to contact the Release Editor to discuss how to get the release note amended.
+
+File naming conventions
+-----------------------
+Release note files need to be named in one of the following ways:-
+
+- Using the Pull Request (PR) number (preferred method)
+- Using the Issue number (to be used if multiple PRs stem from the same issue)
+
+Files need to have unique names. By using either the Issue number or PR number this makes them easier to find if they need to be updated at a later date.
+
+Files should always be in .rst format.
+
+File location
+-------------
+Release note files need to be saved in the directory that best represents their position within the release notes. First identify which release version your note relates to e.g. ``v6.3.0`` . Navigate to this directory and then navigate through the sub-directories until you reach a suitable sub heading. All notes need to be placed within a ``New_features``, ``Improvements``
+or ``Bugfixes`` directory. For example a Bugfix release note for Engineering Diffraction should sit within ``/Diffraction/Engineering/Bugfixes`` .
+
+Release notes should not be placed in any directory outside of ``New_features``, ``Improvements`` or ``Bugfixes`` e.g. do not place release notes in ``/Diffraction/Engineering``. You should also not save release notes in any directory titled ``Used`` as this is for notes that have already been collated into the release notes.
+
+If you are uncertain where your release note should be see the :ref:`Standard File Structure <ReleaseNoteFileStructure>`.
+
+Adding sub-headings
+-------------------
+There will be occasions when the range of standard headings is not suitable for your needs. If you want to add a heading you need to do the following:-
+
+- Add a new directory into the correct part of the filing system. E.g. to add Algorithms to Workbench create the directory ``/Workbench/Algorithm``. The directory name should not contain any spaces.
+- Add ``New_features``, ``Improvements`` and ``Bugfixes`` directories within your new directory. The automated script only works with these directories.
```

Comment:
You can add sub directories but those directories must contain the New_features, Improvements and Bugfixes directories within it. The script only works if the final layer of directories is these 3 on any given branch. I've tried to convey that in this section but maybe thats not clear?

Reply:
That was clear, I wondered if you could add sub-directories to the New_features, Improvements and Bugfixes sub-directories (again, not sure if this would ever be necessary).

Reply:
Any sub directories added to New_features or Bugfixes would not be picked up by the script so these release notes would be effectively invisible. 

Having reviewed this I'm removing Improvements and leaving it as New_features and Bugfixes as it is much simpler going forward (after all what is the difference between a new feature and an improvement?).

Reply:
I think this is the only section that isn't explicitly clear in my mind. Perhaps due to the terms "heading" and "sub-heading" being used interchangeably?

Perhaps make it clear that 
```
* Diffraction **<-This is a heading**

  - Powder Diffraction **<-This is a sub-heading**

	  + New features
	  + Bugfixes

  - Engineering Diffraction

	  + New features
	  + Bugfixes

  - Powder Diffraction

	  + New features
	  + Bugfixes
```

You **cannot** add new headings, you **can** add new sub-headings, you **cannot** add further layers of headings to extend this structure (eg. sub-sub headings).

If I've interpreted that correctly?